scorm-tool/src/java/org/sakaiproject/scorm/ui/upload/UploadPage.java

```diff
@@ -111,9 +111,9 @@ public FileUploadForm( String id )
             add( fileUploadField );
 
             // SCO-98 - disable buttons on submit, add spinner
-            final CancelButton btnCancel = new CancelButton( "btnCancel", PackageListPage.class );
+            final SakaiAjaxCancelButton btnCancel = new SakaiAjaxCancelButton( "btnCancel", PackageListPage.class );
             btnCancel.setOutputMarkupId( true );
-            IndicatingAjaxButton btnSubmit = new IndicatingAjaxButton( "btnSubmit", this )
+            SakaiAjaxButton btnSubmit = new SakaiAjaxButton( "btnSubmit", this )
```

Comment:
The part where it disables itself in updateAjaxAttributes is no longer needs, SakaiAjaxButton should already handle this.